client/src/lib/constants.ts

```diff
@@ -8,9 +8,26 @@ export const SESSION_KEYS = {
   CLIENT_INFORMATION: "mcp_client_information",
 } as const;
 
+export type ConnectionStatus =
+  | "disconnected"
+  | "connected"
+  | "error"
+  | "error-connecting-to-proxy";
+
+export const DEFAULT_MCP_PROXY_LISTEN_PORT = "6277";
```

Comment:
Ideally, this should have been in a common place, where both `server` and `client` access it, will do it in some later PRs.